astro/src/content/docs/get-started/start-here/index.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Hands-on Tutorial
+description: Curious about what FusionAuth can do? Start here to learn about it.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/synedra/fusionauth-first-steps-launch/main/
+
+---
+import ListHostedLoginPagesUseCases from 'src/content/docs/_shared/_list-hosted-login-pages-use-cases.mdx';
+import StartDiagram from 'src/diagrams/docs/get-started/start-here/launch-delegate-add.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import DockerComposeFiles from 'src/content/docs/get-started/download-and-install/_docker-compose-files.mdx';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import DockerSpinup from 'src/components/quickstarts/DockerSpinup.astro';
+import DockerRequirements from 'src/components/quickstarts/DockerRequirements.astro'
+import DockerInstallAndStart from 'src/components/quickstarts/DockerInstallAndStart.astro'
+import GitPodStart from 'src/components/quickstarts/GitPodStart.astro'
+
+To get started with FusionAuth, you need to do three things:
+
+* Launch FusionAuth
+* Delegate authentication
+* Add functionality to your application
+
+<StartDiagram alt="Launch, then delegate, then add functionality." />
+
+Let's walk through an example where you'll add a Login With Google button to an application with no code changes required.
+
+## Prerequisites
+
+* git
```

Comment:
Do we need node installed? Or should we have the sample app run in docker as well (in a node container)?

Reply:
Oh, I think node can be required for Start Here, although adding the sample app as a docker container would be hella sweet.  Maybe a stretch goal :-)